src/content/en/android/trusted-web-activity/whats-new/index.md

```diff
@@ -0,0 +1,461 @@
+
+<div class="clearfix"></div>
+
+Here you’ll find the option to view your existing in-app products and subscriptions and you’ll also
+find the create button for adding new ones.
+
+<!-- Add Image here -->
+<figure class="center">
+  <img src="/web/android/trusted-web-activity/whats-new/in-app-products.jpg"
+      alt="In-app Products"/>
+  <figcaption>
+    <b>Figure 2.</b> In-app Products.
+  </figcaption>
+</figure>
+
+<figure class="attempt-right">
+  <img src="/web/android/trusted-web-activity/whats-new/product-details.jpg"
+      alt="Product Details"/>
+  <figcaption>
+    <b>Figure 3.</b> Product Details.
+  </figcaption>
+</figure>
+
+To create a new in-app product you will need a product ID, name, description, and a price. It’s
+important to create meaningful and easy to remember product IDs, you’ll need them later and the IDs
+can’t be changed once created.
+
+When creating subscriptions you will also have to specify a billing period. You have the option to
+list your subscription benefits and add features like whether you have a free trial, an
+introductory price, a grace period, and a resubscribe option.
+
+After creating each product, make them available from your app by activating them.
+
+If you prefer you can add your products via the [Play Developers API][15].
+
+<div class="clearfix"></div>
+
+Once your catalog is configured, the next step is to configure the checkout flow from the PWA. You
+will use a combination of the [Digital Goods API][12] and the [Payment Request API][11] to achieve
+this.
+
+#### Fetch a product price with the Digital Goods API:
+
+When using Google Play Billing, you will want to ensure that the price displayed to users matches
+the price from the store listing. Manually keeping those prices in sync would be impossible, so
+the Digital Goods API provides a way for the web application to query the underlying payment
+provider for prices:
+
+```javascript
+// The SKU for the product, as defined in the Play Store interface
+async function populatePrice(sku) {
+  try {
+    // Check if the Digital Goods API is supported by the browser.
+    if (window.getDigitalGoodsService) {
+      // The Digital Goods API can be supported by other Payments provider.
+      // In this case, we're retrieving the Google Play Billing provider.
+      const service =
+          await window.getDigitalGoodsService("https://play.google.com/billing");
+
+      // Fetch product details using the `getDetails()` method.
+      const details = await service.getDetails([sku]);
+
+      if (details.length === 0) {
+        console.log(`Could not get SKU: "${sku}".`);
+        return false;
+      }
+
+      // The details will contain both the price and the currenncy.
+      item = details[0];
+      const value = item.price.value;
+      const currency = item.price.currency;
+
+      const formattedPrice = new Intl.NumberFormat(navigator.language, {
+        style: 'currency', currency: currency }).format(value);
```

Comment:
We could maybe have a comment here explaining why we need `Intl.NumberFormat`. Partially it's good practice, partially it's essential because otherwise the numbers will be "7.50000000" or something like that (and this is *not* a bug on our end).